IOTRiskAssessmentFramework/Results/CalculatedRisksFullFramework.csv

```diff
@@ -1,2 +1,2 @@
-Inherent Risk,25.9987
-Residual Risk,13.8723
+Inherent Risk,33.4538
```

Comment:
We will not need the changes in the Results folder. Right?

Reply:
We don't necessarily need them - but I guess it is okay with keep them as a reference - especially once we make changes.